File: src/net/minecraft/server/v1_8_R3/GenLayerZoomFuzzy.java
package net.minecraft.server.v1_8_R3;

public class GenLayerZoomFuzzy extends GenLayerZoom {

    public GenLayerZoomFuzzy(long i, GenLayer genlayer) {
        super(i, genlayer);
    }

    protected int b(int i, int j, int k, int l) {
        return this.a(new int[] { i, j, k, l});
    }
}

File: Server_Is_NaN/Server_Is_NaN/Server/Worlds/Generator/GenLayers/GenLayerZoomFuzzy.cs
namespace Server_Is_NaN.Server.Worlds.Generator.GenLayers
{
    public class GenLayerZoomFuzzy : GenLayerZoom
    {
        public GenLayerZoomFuzzy(GenLayer parent, long worldSeed) : base(parent, worldSeed)
        {
        }

        protected override int SelectModeOrRandom(int i1, int i2, int i3, int i4)
        {
            return SelectRandom(i1, i2, i3, i4);
        }
    }
}
